docs/values.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: networking.gke.io/v1beta1
```

Comment:
what's this file for? Is it meant to be in the docs directory?

Reply:
This was the example file you would create if you follow the helm install instructions. Thought it might be useful to just have one created as a template? I can remove it if you think it's not needed.